src/server/cmd/pachd/main.go

```diff
@@ -980,6 +988,11 @@ func doFullMode(config interface{}) (retErr error) {
 		g.Wait()
 		log.Println("gRPC server gracefully stopped")
 	}(interruptChan)
+	for _, b := range bootstrappers {
+		if err := b(context.Background()); err != nil {
```

Comment:
It would be nice if we knew and printed which bootstrapper failed here.  Maybe the []bootstrapFunc could be map[string]bootstrapFunc instead?  (Do we care about order?)